src/tcgwars/logic/util/CardTypeSet.java

```diff
@@ -31,7 +33,8 @@ public CardTypeSet(CardType... cardTypes) {
     }
   }
   public boolean is(CardType o){
-    return contains(o);
+    List<PlayerType> Holon_Veil = TcgStatics.bg().em().retrieveObject("Holon_Veil");
+    return (contains(o) || (o == CardType.DELTA && contains(CardType.POKEMON) && Holon_Veil.contains(player)));
```

Comment:
I know that player doesnt exist within cardtype but am unsure how else to access the owner of a card that is not in a pcs